transport-parent/transport-netty/src/main/java/io/scalecube/transport/netty/TransportImpl.java

```diff
@@ -34,6 +36,7 @@
 public final class TransportImpl implements Transport {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(Transport.class);
+  private static final DistinctErrorLog log = new DistinctErrorLog(Duration.ofMinutes(1));
```

Comment:
Plz make new line for this staic variable. Use capital chars for the name. Name it plz `ERROR_LOG`.